probe/base/metrics.go

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2022, MegaEase
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package base
+
+import (
+	"github.com/megaease/easeprobe/global"
+	"github.com/megaease/easeprobe/metric"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// Metrics is the probe metrics
+type Metrics struct {
```

Comment:
Metrics struct just be referred in the package, so rename to `metrics` is better? Name of the fields should be renamed too?

Reply:
I think renaming the `Metrics` =>`metrics` and  `NewMetric()` => `newMetrics()`should be fine.